src/forwarder/miniforwarder.cpp

```diff
@@ -113,8 +116,10 @@ miniforwarder::miniforwarder(const std::string& config,
 
     _db = std::unique_ptr<Scoreboard>(
             new Scoreboard(redis_host, redis_port, redis_db, redis_pwd));
-    _daq = std::unique_ptr<DAQFetcher>(new DAQFetcher(_partition.c_str()));
+    _daq = std::unique_ptr<DAQFetcher>(new DAQFetcher(_partition.c_str(),
+                segment_order));
     _sender = std::unique_ptr<FileSender>(new FileSender(xfer_option));
+    _fmt = std::unique_ptr<FitsFormatter>(new FitsFormatter(segment_order));
 
     _forwarder_list = "forwarder_list";
     _association_key = "f99_association";
```

Comment:
This should be a configuration file entry.

Reply:
I was planning to use UUID or some kind of unique ID that is autogenerated.